addons/web/static/src/core/pager/pager_indicator.scss

```diff
@@ -1,6 +1,16 @@
 .o_pager_indicator {
     z-index: $zindex-modal + 1;
     transition: opacity 0.4s;
+
+    > .o_pager_indicator_inner {
+        min-width: 13ch;
+        padding: $o-touch-btn-padding;
+        text-align: center;
```

Comment:
Can we use bootstrap classes when possible?
- Do we absolutely need to use a $o-touch-btn-padding
- Same for the bg color, radius and border